tests/src/test/scala/quasiquotes/ErrorSuite.scala

```diff
@@ -297,9 +297,11 @@ class ErrorSuite extends FunSuite {
       val tpes = List(q"T")
       q"expr: ..$tpes"
     """) === """
-      |<macro>:5: identifier expected but ellipsis found
+      |<macro>:5: type mismatch when unquoting;
+      | found   : List[scala.meta.Term.Name]
+      | required: scala.collection.immutable.Seq[scala.meta.Type]
```

Comment:
I think the new message comes from the fact that you changed `name` to accept ellipses in addition to accepting unquotes. Why was that necessary?